docs/cpp/configure-intellisense.md

```diff
@@ -4,63 +4,67 @@ Area: cpp
 
-![configure-intelliSense-indicator-status-bar](images/intellisense/configure-intellisense-indicator.png)
+![Configure IntelliSense indicator in the status bar](images/intellisense/configure-intellisense-indicator.png)
 
-To configure, select the status bar indicator, which navigates you to the [configuration quick pick](/docs/cpp/configure-intellisense.md#_Option-1.-Select-a-configuration-option-through-the-configuration-quick-pick). The quick pick can help you select or install a C/C++ compiler.
+To configure, select the status bar indicator, which brings up the [configuration Quick Pick](#option-1-select-a-configuration-option-through-the-configuration-quick-pick). The Quick Pick can help you select or install a C/C++ compiler.
 
-If you do not see a status bar indicator, you can also check your project's `c_cpp_properties.json` file. This file stores all of your IntelliSense configuration settings. Navigate to this file by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`). Check the `IntelliSense mode` to find your configuration. To learn more about the `c_cpp_properties.json` file, see the [schema reference](/docs/cpp/c-cpp-properties-schema-reference.md).
+If you do not see a status bar indicator, you can also check your project's `c_cpp_properties.json` file. This file stores all of your IntelliSense configuration settings. Navigate to this file by selecting **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`). Check the `IntelliSense mode` to find your configuration. To learn more about the `c_cpp_properties.json` file, you can review the [schema reference](/docs/cpp/c-cpp-properties-schema-reference.md).
 
-![Command Palette](images/cpp/command-palette.png)
+![Command Palette filtered on C/C++ extension commands](images/cpp/command-palette.png)
 
 ## How to configure IntelliSense
 
 IntelliSense configuration is stored in the `c_cpp_properties.json` file, which is automatically created in your workspace. All three of the following options are different ways of editing the `c_cpp_properties.json` file:
 
-### Option 1. Select a configuration option through the configuration quick pick
+### Option 1. Select a configuration option through the configuration Quick Pick
 
-Open the quick pick by entering **Select IntelliSense Configuration** in the Command Palette (`kb(workbench.action.showCommands)`), which shows you a dropdown with all of the configuration options found by the C/C++ Extension on your machine.
+Open the Quick Pick by entering **Select IntelliSense Configuration** in the Command Palette (`kb(workbench.action.showCommands)`), which shows you a dropdown with all of the configuration options found by the C/C++ extension on your machine.
 
-![Compiler Quick Pick](images/intellisense/compiler-quick-pick.png)
+![Select a compiler to configure for IntelliSense Quick Pick](images/intellisense/compiler-quick-pick.png)
```

Comment:
Same here, we can make this shorter with this image: 
![image](https://github.com/microsoft/vscode-docs/assets/67650598/e6e95221-5bb6-4b39-bbde-5301fc5fc687)